src/gbd_mapping_generator/util.py

```diff
@@ -28,7 +28,11 @@ def make_import(module_to_import, imports=()):
 
 def text_wrap(start_string, items, sep=', ', implicit=False):
     if len(start_string + sep.join(items)) <= TEXTWIDTH:
-        out = start_string + sep.join(items) + '\n'
+        out = start_string
```

Comment:
this adds a lot of parentheses where you don't want them and doesn't fix your problem